src/sections/InvolvementSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
I think this component is slightly bigger than viewport width and it's creating a horizontal scrollbar on the page